src/lib.rs

```diff
@@ -858,6 +858,11 @@ pub unsafe trait FromZeroes {
     }
 }
 
+/// Deprecated: prefer [`FromZeros`] instead.
+#[deprecated(since = "0.7.6", note = "FromZeroes has been renamed to FromZeros")]
```

Comment:
Deprecating is a breaking change for users who deny warnings in their dependencies; we need to wait for 0.8 to mark this as deprecated.